lib/ibanity/util/datetime_util.ex

```diff
@@ -3,8 +3,11 @@ defmodule Ibanity.DateTimeUtil do
 
   def parse(string) do
     case DateTime.from_iso8601(string) do
-      {:ok, datetime, 0} -> datetime
-      {:error, _} -> nil
+      {:ok, datetime, 0} ->
+        datetime
+
+      {:error, _} ->
```

Comment:
Indeed, and that's the point: instead of silently return nil - and then break elsewhere when `nil` is used instead of a value - it raises an exception telling exactly where the problem is.

Reply:
The behaviour change has been described in the Changelog